tests/core/test_tracker_stores.py

```diff
@@ -125,6 +126,23 @@ def test_tracker_store_from_invalid_string(default_domain):
     assert isinstance(tracker_store, InMemoryTrackerStore)
 
 
+def test_tracker_serialisation():
+    slot_key = "location"
+    slot_val = "Easter Island"
+
+    store = InMemoryTrackerStore(domain)
+
+    tracker = store.get_or_create_tracker(UserMessage.DEFAULT_SENDER_ID)
+    ev = SlotSet(slot_key, slot_val)
+    tracker.update(ev)
+
+    serialised = store.serialise_tracker(tracker)
```

Comment:
can we also add a test for the conversation dumping?

Reply:
hm, the serialisation already dumps the conversation. or what do you mean?

Reply:
no idea what I meant 👀